packages/layout/stories/link.stories.tsx

```diff
@@ -31,3 +31,12 @@ export const WithRoutingLibrary = () => (
     </Link>
   </BrowserRouter>
 )
+
+/**
+ * This is a link in a disabled state.
+ */
+export const WithDisabledState = () => (
+  <Link isExternal href="google.com" isDisabled>
```

Comment:
I believe this will result in a relative url in storybook, testing with `https://` might be better, unsure though :)

Reply:
Yes exactly, I will also update the other Link examples to then, because they also have it like that.